frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js

```diff
@@ -682,6 +689,15 @@ export default function InteractiveWaveformModal({
   };
 
   const handleVersionSelect = (idx) => {
+    const selectedVersionMetaData = metaPulse3dVersions.filter(
```

Comment:
Metadata is one word so this should be `selectedVersionMetadata`, the 'd' in data does not need to be capitalized